tests/jeos/glibc_locale.pm

```diff
@@ -233,7 +234,7 @@ sub run {
     my ($self) = @_;
     # C<$lang_ref> denotes what kind of lang setting is expected from test suite perspective
     # sle15+ does not enable locale change during firstboot
-    my $lang_ref = get_var('JEOSINSTLANG', 'en_US');
+    my $lang_ref = is_translations_preinstalled() ? get_var('JEOSINSTLANG', 'en_US') : 'en_US';
```

Comment:
This is most likely not needed, because of the translations are not preinstalled, hence sut is not 12sp5, this module would install them. Since 12sp5 is the only one with pre-installed translations are will go away soon. We should not introduce any dependencies in this regard.

Thus, the minimal locale testing, meaning asserting correct choice in jeos-firstboot should be done in firstrun module.